domains/colaslush.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jordan240",
+        "email": "jordanmolina2022@gmail.com"
+    },
+    "records": {
+        "A": "216.198.79.1"
```

```suggestion
        "CNAME": "cname.vercel-dns.com"
```

If you want to provide other records such as MX or TXT, remove the CNAME and use the following record instead:
```suggestion
        "A": ["216.198.79.1"]
```